### PR TITLE
fwupd(-efi), libjcat, flashrom, gnu-efi-libs update and compatibility checks.

### PR DESCRIPTION
**Pull request by dataCobra**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x86_64-musl
  - i686
  - aarch64 (cross)
  - armv7l (cross)

`fwupd` does need the new version of `libjcat`.

Since `libjcat` is only used for `fwupd` I put both updates into this PR.

### CODE REVIEW

**Comment by dkwo** [comment]
Hey, do you mind also updating `fwupd-efi`?

**Comment by dataCobra** [comment]
> Hey, do you mind also updating `fwupd-efi`?

I can also update `fwupd-efi` but it seems not to be trivial so I would create a separate PR for that.

**Comment by dkwo** [comment]
it may not be necessary anymore to carry the patch disabling test: that test has been moved to test-slow, which should be disabled by default (at least in qemu). can you try dropping it?

**Comment by dkwo** [comment]
Also, v2.0 is out with many changes in case you're interested.

**Comment by dataCobra** [comment]
Currently in the work of getting 2.0 to work.

**Comment by dataCobra** [comment]
The major update does need way more tinkering and testing.

We should update to this version while I'm still working on the big update.

**Comment by dataCobra** [comment]
To make sure everything works together as intended I've updates this PR.

**Comment by dataCobra** [comment]
Tested the whole functionality successful on a Laptop.

:white_check_mark: -> fwupd
:whi

File: srcpkgs/fwupd/template [hunk 1]
```diff
@@ -12,8 +12,9 @@ hostmakedepends="dejavu-fonts-ttf gnutls-tools pkg-config gcab gi-docgen vala
  protobuf-c-devel"
 makedepends="libxmlb-devel cairo-devel colord-devel libarchive-devel
  gnutls-devel gpgme-devel json-glib-devel libgusb-devel polkit-devel
```
Comment (leahneukirchen) [inline srcpkgs/fwupd/template:14]: libgusb-devel can be dropped i think

Comment (dataCobra) [reply srcpkgs/fwupd/template]: Packaging worked indeed fine without the `libgusb-devel` makedepends.

**Pull request merged**